o.js

```diff
@@ -100,14 +100,14 @@ o.prototype.add = function(children) {
 };
 
 o.prototype.for = function (id){
-    if (this.element.nodeName === 'label'){
+    if (this.element.nodeName === 'LABEL'){
         this.element.setAttribute('for', id);
     }
     return this;
 };
 
 o.prototype.text = function(text) {
-    if (typeof(text) != 'undefined' && text != 'undefined') {
+    if (typeof(text) !== undefined && text !== '') {
```

Comment:
1) nie wiedziałem, że typeof można też używać jak funkcji :D Ciekawostka: spróbuj sobie typeof(typeof) 
2) Dlaczego nie można tu podać pustego stringa? 
3) Bardziej sprawdziłbym czy nie jest undefined/object/function. Może taki myczek na przykład:
```suggestion
    if (!['undefined', 'object', 'function'].includes(typeof text)) {
```

Reply:
Tak, dobra uwaga. Tylko stylistycznie chciałem to poprawić jak scrollowałem i pomyślałem, ze po co nie można wpisać undefined i zmieniłem na pusty string. Commituje sugestie